ftw/publisher/sender/utils.py

```diff
@@ -136,8 +136,13 @@ def get_site_relative_path(obj):
     obj_path = '/'.join(obj.getPhysicalPath())
     portal_path = '/'.join(portal.getPhysicalPath())
 
+    path = obj_path[len(portal_path):]
```

Comment:
Since we have quite a lot of paths in this method, I'd be more explicit, maybe name it `rel_path` or even `rel_obj_path`.